### ISSUE TITLE
Remote Monitor/Laptop

### ISSUE THREAD

**Issue by adamos42** [slgobinath/SafeEyes#14]
I can see that #6 will add multi monitor support and I want to expand that to my laptop which are next to my two monitor. I want some sort of solution what Synergy has, so I could configure a client and server and the server would tell the client that is now a rest time.

**Comment by slgobinath** [comment]
That's a good idea. I will consider this after implementing https://github.com/slgobinath/SafeEyes/issues/6.

**Comment by nifadyev** [comment]
Hey @adaliszk , Is the issue still relevant to you?
I suppose this feature request is not planned in near future since it will require a lot of new logic.
@deltragon , @archisman-panigrahi , what do you think?

**Comment by adaliszk** [comment]
Yes, it's still relevant, albeit my setup has changed to a single monitor with a split view. 
Essentially, one side of the monitor does lock up while the other remains active.

Now, a simple way that could help is by triggering a keystroke when locking the session for a break; that way, I can define that keystroke in Synergy to relay a lock action on all connected machines.
